Classes/Utility/ContentObjectFetcher.php

```diff
@@ -22,23 +23,29 @@ public static function resolve(?ConfigurationManagerInterface $configurationMana
             ? $configurationManager->getRequest()
             : ($GLOBALS['TYPO3_REQUEST'] ?? null);
 
-        if ($request && $configurationManager === null) {
+        if ($request
```

Comment:
We can simplify this to just if-request-then-fetch-from-request.

The code below then only enters the fetch-from-configurationmanager case if it wasn't already found in request. So this should give the same result, but gives preference to Request instead of ConfigurationManager on pre-v12 (which should be fine).

Reply:
Adjusted.